includes/class-sensei-content-drip.php

```diff
@@ -274,10 +276,34 @@ public function __wakeup() {
 	 * Installation. Runs on activation.
 	 *
 	 * @since  1.0.0
+	 * @access private
 	 * @return void
 	 */
 	public function install() {
 		$this->_log_version_number();
+
+		$hook = 'woo_scd_daily_cron_hook';
+
+		if ( false !== wp_next_scheduled( $hook ) ) {
+			wp_clear_scheduled_hook( $hook );
+		}
+
+		$today_start         = strtotime( date_i18n( 'Y-m-d' ) );
+		$tomorrow_start      = $today_start + 24 * HOUR_IN_SECONDS;
+		$scheduled_time      = $tomorrow_start + 30 * MINUTE_IN_SECONDS;
+		$scheduled_time_unix = $scheduled_time - get_option( 'gmt_offset' ) * HOUR_IN_SECONDS;
+		wp_schedule_event( $scheduled_time_unix, 'daily', $hook );
+	}
+
+	/**
+	 * Runs on deactivation.
+	 *
+	 * @since 2.0.0
+	 * @access private
+	 */
+	public function on_deactivation() {
```

Comment:
`on_deactivation` sounds like an event name. `uninstall` on deactivation matches the `install` on activation. Alternatively, `activate` and `deactivate` work as well.

Reply:
https://github.com/woocommerce/sensei-content-drip/pull/160/commits/d8675bf8a7b9dd19aa4f41a1fdecd317e85e42ea